cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/task/LoadMonitorTaskRunnerTest.java

```diff
@@ -214,7 +214,7 @@ public Samples getSamples(Cluster cluster,
         PartitionMetricSample sample = new PartitionMetricSample(cluster.partition(tp).leader().id(), tp);
         long now = TIME.milliseconds();
         for (Resource resource : Resource.values()) {
-          sample.record(KafkaMetricDef.resourceToMetricInfo(resource), now);
+          sample.record(KafkaMetricDef.resourceToMetricInfo(resource).get(0), now);
```

Comment:
Can we maybe add a function as a wrapper to `.get(0)` call? It is not obvious why we get the value in index 0.

Reply:
Doesn't really matter here. I added the values for all the metrics of a resource.